docs/sources/installation/istio.md

```diff
@@ -0,0 +1,179 @@
+# Installation instructions for Istio
+
+The ingestor, querier, etc. might start, but if those changes are not made, you will see logs like
```

Comment:
When installing Loki on Istio service mesh you must complete some additional steps.  Without these steps, the ingestor, querier, etc. might start, but you will see logs like the following: